include/class.organization.php

```diff
@@ -425,6 +425,11 @@ function update($vars, &$errors) {
                 $u->setPrimaryContact(array_search($u->id, $vars['contacts']) !== false);
                 $u->save();
             }
+        } else {
+            foreach ($this->allMembers() as $u) {
```

Comment:
Use `ORM` mass update instead of `foreach` loop.
```php
$members = $this->allMembers();
return $members->update(array(
   'flags' => SqlExpression::bitand(
        new SqlField('flags'), ~User::PRIMARY_ORG_CONTACT)
   ));
```